src/main/java/com/example/syncrowbackend/friendrequestroom/dto/PostDto.java

```diff
@@ -0,0 +1,43 @@
+package com.example.syncrowbackend.friendrequestroom.dto;
+
+import com.example.syncrowbackend.friendrequestroom.entity.Post;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class PostDto {
+    private Long id;
+    @NotBlank(message = "게시물에 제목은 필요합니다.")
+    private String title;
+    private String content;
+    private String username;
+    private String profileImage;
+    private Double temp;
+
+    @Builder.Default
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    private LocalDateTime createdAt = LocalDateTime.now();
+
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    LocalDateTime modifiedAt;
+
+    @Builder
+    public static PostDto toDto(Post post) {
+        return PostDto.builder()
+                .id(post.getId())
+                .username(post.getUser().getUsername())
+                .title(post.getTitle())
+                .content(post.getContent())
+                .modifiedAt(LocalDateTime.now())
```

Comment:
```suggestion
                .modifiedAt(post.getModifiedAt())
```
엔티티에 적용된 auditing 기능을 활용하는게 어떨까요 ??

Reply:
아하 저 부분 수정을 안했네요😅 피드백 감사합니다!!